src/common_lib/include/common_lib/structures/angle_and_norms.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef SRC_COMMON_LIB_INCLUDE_COMMON_LIB_STRUCTURES_ANGLE_AND_NORMS_HPP_
+#define SRC_COMMON_LIB_INCLUDE_COMMON_LIB_STRUCTURES_ANGLE_AND_NORMS_HPP_
+
```

Comment:
This does not necessarily need to be in structures for being a structure. It is ok but if you have that opinion I think it would be fine in maths file too

Reply:
Agree with Marcelo